html/janus.js

```diff
@@ -1429,13 +1438,12 @@ function Janus(gatewayCallbacks) {
 			// If we got here, we're not screensharing
 			if(media === null || media === undefined || media.video !== 'screen') {
 				// Check whether all media sources are actually available or not
-				// as per https://github.com/meetecho/janus-gateway/pull/114
-				MediaStreamTrack.getSources(function(sources) {
-					var audioExist = sources.some(function(source) {
-						return source.kind === 'audio';
+				navigator.mediaDevices.enumerateDevices().then(function(devices) {
```

Comment:
Mh, replacing adapter.js is a bit risky as we had some custom changes there, especially with respect to Edge support that this might currently break... I'll have to look into this.